presto-main/src/main/java/com/facebook/presto/server/protocol/QueuedStatementResource.java

```diff
@@ -166,7 +171,7 @@ public Response postStatement(
             throw badRequest(BAD_REQUEST, "SQL statement is empty");
         }
 
-        SessionContext sessionContext = new HttpRequestSessionContext(servletRequest);
+        SessionContext sessionContext = new HttpRequestSessionContext(servletRequest, this.sqlParserOptions);
```

Comment:
nit: we only use the "this" prefix in the constructor.  Otherwise just do sqlParserOptions